SPHINXsys/src/shared/particles/base_particles.cpp

```diff
@@ -289,7 +288,7 @@ namespace SPH
 			output_file << std::endl;
 			output_file << "    </DataArray>\n";
 		}
-		/*
+		
```

Comment:
is it a comment?

Reply:
I commented this part out temporarily. We need to restructure this - Mariia already started with it